docs/snippets/_mobile_docs_prebid-mobile.mdx

```diff
@@ -0,0 +1,114 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This section is for participants who want to integrate with EUID and use the [Prebid Mobile SDK](https://prebid.org/product-suite/prebid-mobile/) for header bidding in Android and iOS applications.
+
+The EUID Prebid integration monitors the state of the `UID2Identity`. Whenever the state changes, the Prebid integration automatically updates Prebid’s external user IDs. This includes when an identity is refreshed, resulting in a new advertising token.
+
+Prebid then sends the EUID tokens into the RTB bidstream, along with other external user IDs that you might set.
+
+:::note
+This integration requires a Prebid Server setup.
+:::
+
+To configure your EUID Prebid for Mobile integration, follow these steps:
+
+1. Set up Prebid's Mobile SDK, following the steps in [Prebid SDK Integration for Android](https://docs.prebid.org/prebid-mobile/pbm-api/android/code-integration-android.html) or [Prebid SDK Integration for iOS](https://docs.prebid.org/prebid-mobile/pbm-api/ios/code-integration-ios.html).
+
+2. Add the EUID Mobile SDK to your app, following the steps in [Add the EUID Mobile SDK to Your Mobile App](#add-the-euid-mobile-sdk-to-your-mobile-app).
+
+3. The EUID Prebid integration is distributed as a separate module, so you must add it as a dependency in your project. Follow the installation instructions that apply to your integration, out of the following options:
+
+   <Tabs groupId="dependency-selection">
+   <TabItem value='gradle' label='Gradle (Android)'>
+   Include the following in your Gradle configuration:
+   ```js
+   implementation("com.uid2:uid2-android-sdk-prebid:1.4.0")
+   ```
+   </TabItem>
+   <TabItem value='maven' label='Maven (Android)'>
+   To install with Maven, add the SDK as a dependency in the `pom.xml` file:
+   ```xml
+   <dependency>
+      <groupId>com.uid2</groupId>
+      <artifactId>uid2-android-sdk-prebid</artifactId>
+      <version>1.4.0</version>
```

Comment:
android version will be 1.5.0

Reply:
@caroline-ttd Done. Updated 2 instances in this file. Please check.